Comp2023/src/main/java/frc/robot/subsystems/Wrist.java

```diff
@@ -323,6 +323,16 @@ private void wristTalonInitialize(WPI_TalonFX motor, boolean inverted)
   public void moveWristWithJoystick(XboxController joystick)
   {
     double axisValue = -joystick.getRightY( );
+    moveWristInput(axisValue);
+  }
+
+  public void moveWristConstantSpeed(double speed)
```

Comment:
Not really entirely happy with the method names being clear enough, but they are all GOOD ENOUGH for getting through MBR.